drivers/soundwire/dmi-quirks.c

```diff
@@ -73,6 +73,23 @@ static const struct adr_remap hp_omen_16[] = {
 	{}
 };
 
+/*
```

Comment:
It is better to use `Link` instead of `BugLink:` in commit message

Reply:
I see, I've updated the commit messages.